src/balancereader/src/main/resources/log4j2.xml

```diff
@@ -23,6 +23,9 @@
         </Console>
     </Appenders>
     <Loggers>
+        <Logger name="org.hibernate" level="WARN" additivity="false">
```

Comment:
based on this line add, I'm guessing we weren't logging hibernate plugin logs before this? is this to log if any metrics error happens?

Reply:
Yes, we were previously not logging the hibernate plugin logs, and this would log any errors in the hibernate plugin (if an entity didnt load properly or if metrics error)